src/Symfony/Component/Translation/Resources/functions/translatable.php

```diff
@@ -1,22 +0,0 @@
-<?php
-
-/*
- * This file is part of the Symfony package.
- *
- * (c) Fabien Potencier <fabien@symfony.com>
- *
- * For the full copyright and license information, please view the LICENSE
- * file that was distributed with this source code.
- */
-
-use Symfony\Component\Translation\Translatable;
-
-if (!function_exists('t')) {
```

Comment:
removing this `function_exists` can cause conflicts with other projects (this function is not namespaced)

Reply:
Forget about that one. You added a namespace in parallel of my review...